opentelemetry-sdk/src/opentelemetry/sdk/_metrics/__init__.py

```diff
@@ -64,7 +64,8 @@ def __init__(
         self._instrumentation_info = instrumentation_info
         self._measurement_consumer = measurement_consumer
 
-    def create_counter(self, name, unit=None, description=None) -> APICounter:
+    def create_counter(self, name, unit="", description="") -> APICounter:
+        self._check_instrument_id(name, Counter, unit, description)
```

Comment:
I know you said you would address it in a separate PR, but how do you plan to suppress logging a warning here if there is a view which addresses the conflict?

Reply:
Right, separated the checking from the warning logging so that the second step can be done after a view check in a subsequent PR.